raiden/network/rpc/client.py

```diff
@@ -509,18 +509,18 @@ def __init__(
 
         result = jsonrpc_client.call(
             'eth_getCode',
-            address_encoder(token_address),
+            token_address,
```

Comment:
I found out whats the problem with this, there is a different encoding through the tests, so I am fixing that.